default-placeholders/index.md

```diff
@@ -162,19 +205,29 @@ class accessor {
   
   // 2
   // Only available when: ((accessTarget == access::target::global_buffer) ||
-  //                       (accessTarget == access::target::constant_buffer) ||
-  //                       (accessTarget == access::target::host_buffer)) &&
+  //                       (accessTarget == access::target::constant_buffer)) &&
   //                      (dimensions == 0)
   accessor(buffer<dataT, 1> &bufferRef);
 
   // 3
   // Only available when: ((accessTarget == access::target::global_buffer) ||
-  //                       (accessTarget == access::target::constant_buffer) ||
-  //                       (accessTarget == access::target::host_buffer)) &&
+  //                       (accessTarget == access::target::constant_buffer)) &&
   //                      (dimensions > 0)
   accessor(buffer<dataT, dimensions> &bufferRef,
            range<dimensions> accessRange,
            id<dimensions> accessOffset = {});
+
+  // 4
+  // Only available when (accessTarget == access::target::host_buffer) &&
+  //                     ((otherTarget == access::target::global_buffer) ||
+  //                      (otherTarget == access::target::constant_buffer))
+  template <access::target otherTarget, access::placeholder otherPlaceholder>
+  accessor(accessor<dataT,
```

Comment:
I would prefer that we make this a member function or make the constructor explicit, just to make the act of creating a host accessor from a device accessor in-place more explicit.

Reply:
Done, I've also made the constructor from a buffer explicit